app/src/main/java/woowacourse/payments/ui/screen/registration/CardRegistrationActivity.kt

```diff
@@ -0,0 +1,33 @@
+package woowacourse.payments.ui.screen.registration
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import woowacourse.payments.ui.theme.AndroidpaymentsTheme
+
+class CardRegistrationActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContent {
+            AndroidpaymentsTheme {
+                CardRegistrationScreen(
+                    onBackClick = { finish() },
```

Comment:
```suggestion
                    onBackClick = ::finish,
```

위와 같이 고쳐볼 수도 있겠네요. 👍